hack/e2e-test.sh

```diff
@@ -0,0 +1,28 @@
+export KUSTOMIZE=$PWD/bin/kustomize
+export GINKGO=$PWD/bin/ginkgo
+export KIND=$PWD/bin/kind
+function cleanup {
+    if [ $USE_EXISTING_CLUSTER == 'false' ] 
+    then 
+        $KIND delete cluster --name $KIND_CLUSTER_NAME
+    fi
+    (cd config/manager && $KUSTOMIZE edit set image controller=gcr.io/k8s-staging-jobset/jobset:main)
+}
+function startup {
+    if [ $USE_EXISTING_CLUSTER == 'false' ] 
+    then 
+        $KIND create cluster --name $KIND_CLUSTER_NAME --image $E2E_KIND_VERSION
+    fi
+}
+function kind_load {
+    $KIND load docker-image $IMAGE_TAG --name $KIND_CLUSTER_NAME
+}
+function jobset_deploy {
+    (cd config/manager && $KUSTOMIZE edit set image controller=$IMAGE_TAG)
```

Comment:
can you make the shell script print every command it executes so we know what is going on?